README.md

```diff
@@ -3,6 +3,8 @@
 [![MELPA](https://melpa.org/packages/olivetti-badge.svg)](https://melpa.org/#/olivetti)
 
 Olivetti is a simple Emacs minor mode for a nice writing environment.
+Like the famous typewriter, Olivetti mode lets you focus on writing
+rather than on the interface.
```

Comment:
Maybe let’s remove this bit though and I’ll add something in response to your comment in #18